### PR TITLE
Controller specs

### PR DESCRIPTION
**Pull request by Fadahunsiseyi**
### In the MileStone below, I completed the following;

-  Add RSpec as a gem to your project.

- Create a Request spec file for all your controllers.

- Make sure that for each action you check:

- If response status was correct.

- If a correct template was rendered.

- If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    it 'should render the  template' do
+      expect(response).to render_template(:index)
+    end
+    it 'return correct response' do
+      expect(response).to have_http_status(:ok)
+    end
+    it 'body should includes correct placeholder text' do
+      expect(response.body).to include('HELLO RAILS index 2')
+    end
+  end
```
Comment (AnselemOdims) [inline spec/requests/posts_spec.rb]: - [ ] Just as you included a test for the show action in the user controller test, please kindly do the same for the post controller

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Fadahunsiseyi , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: spec/requests/posts_spec.rb (hunk 1 above)
Comment (Fadahunsiseyi) [reply spec/requests/posts_spec.rb]: @AnselemOdims I have implemented the change required above.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Fadahunsi, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**